subsys/bluetooth/audio/Kconfig.mediacontrol

```diff
@@ -0,0 +1,50 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+config BT_MEDIACONTROL
+       bool "Support for media player and media player control"
+       help
+         Enables support for media player and for control of local and remote
```

Comment:
```suggestion	
	 Enables support for media player and for control of local and remote
```

Reply:
Same goes further down it seems

Reply:
It seems you want a white space change - but it is hard to see what the change is.

Reply:
Tabs instead of spaces, I suspect now

Reply:
Yes, sorry - That certainly isn't clear from just looking at it :D